### DISCUSSION TITLE
Issues are automatically deleted

### DISCUSSION THREAD

**Discussion by JonasSchaber** [upptime/upptime#510] [Q&A] [answered]
Hey, I've been using Upptime for a few weeks now. A few days ago it happened for the first time that an issue was opened automatically and then closed and deleted again. I read here in the Q&A that this is because they are false positives, right?

If this is the case, is there a way to get around this automatic deletion of issues?

Thanks in advance for an answer!

**Comment by vichaos** [comment]
I'm facing this issue too. No new incident report in the "Past Incidents" section because the issue has been deleted.

**Comment by AnandChowdhary** [comment, marked as answer]
Please see https://github.com/upptime/upptime/discussions/492
